strongbox-aql/src/main/java/org/carlspring/strongbox/services/impl/AqlSearchServiceImpl.java

```diff
@@ -30,7 +29,7 @@
 public class AqlSearchServiceImpl implements AqlSearchService
 {
 
-    @PersistenceContext
+    //@PersistenceContext
```

Comment:
Is this dead code? If so, could you please remove it?

Reply:
all the AQL related code should be reworked, this is just as a reminder that it was the `PersistenceContext` before.

Reply:
Okay, thanks! :)